config/initializers/content_security_policy.rb

```diff
@@ -22,6 +22,8 @@
     '*.nr-data.net',
     'dap.digitalgov.gov',
     '*.google-analytics.com',
+    'h-api.online-metrix.net',
+    'h.online-metrix.net',
```

Comment:
Something I came across in testing is that when `RAILS_ENV` is not `production`, the CSP doesn't get applied. 
If `proofing_device_profiling_collecting_enabled` is `true`, we should include `h.online-metrix.net` in CSP for non-production environments.